README.md

```diff
@@ -11,41 +11,67 @@ using [Syft](https://github.com/anchore/syft).
 
 By default, this action will execute a Syft scan in the workspace directory
 and upload a workflow artifact SBOM in SPDX format. It will also detect
-if being run during a `release` and attach the SBOM
-as a release asset.
+if being run during a [GitHub release](https://docs.github.com/en/repositories/releasing-projects-on-github/about-releases)
+and upload the SBOM as a release asset.
 
 ## Example Usage
 
-### Scan a docker image
+### Scan a container image
 
-Use the `image` parameter
+To scan a container image from the Docker daemon, use the `image` parameter:
 
 ```yaml
 - uses: anchore/sbom-action@main
   with:
     image: example/image_name
 ```
 
+To use a specific container registry, specify a typical _registry path_ to the
+image including the registry host. This will
+attempt to use the Docker daemon and by association any previously configured
+credentials. If the Docker daemon is not available, a direct connection to the
+registry will be attempted:
+
+```yaml
+- uses: anchore/sbom-action@main
+  with:
+    image: ghcr.io/example/image_name:tag
+```
+
+To use an authenticated direct connection to the registry regardless of the
+Docker daemon availability, add credentials using
+`registry-username` and `registry-password`:
+
+```yaml
+- uses: anchore/sbom-action@main
+  with:
+    image: my-registry.com/my/image
+    registry-username: mr_awesome
+    registry-password: ${{ secrets.REGISTRY_PASSWORD }}
+```
+
 ### Scan a specific directory
 
-Use the `path` parameter, relative to the repository root
+Use the `path` parameter, relative to the repository root:
 
 ```yaml
 - uses: anchore/sbom-action@main
   with:
     path: ./build/
 ```
 
-### Attach SBOMs to a release
+### Publishing SBOMs with releases
 
-The action will detect being run in a `release` and
-automatically upload all SBOMs as release assets. However,
+The `sbom-action` will detect being run during a
+[GitHub release](https://docs.github.com/en/repositories/releasing-projects-on-github/about-releases)
+and automatically upload all SBOMs as release assets. However,
 it may be desirable to upload SBOMs generated with other tools or using Syft
-outside of this action. To do this, specify a regular expression using
-the `sbom-artifact-match` pararmeter, for example:
+outside this action. To do this, use the `anchore/sbom-action/publish-sbom` sub-action
+and specify a regular expression with the `sbom-artifact-match`
+parameter:
 
 ```yaml
-- uses: anchore/sbom-action/attach@main
+- uses: anchore/sbom-action/publish-sbom@main
```

Comment:
Note: this includes a couple other doc changes and renames:

`/attach` -> `/publish-sbom`
`/download` -> `/download-syft`

etc.

Reply:
I like these changes!